lib/testing.dart

```diff
@@ -11,14 +11,15 @@
 ///     import 'dart:convert';
 ///     import 'package:http/testing.dart';
 ///
-///     var client = new MockClient((request) {
+///     var client = new MockClient((request) async {
```

Comment:
It doesn't type check otherwise. The expected type of the constructor argument is
```dart
typedef Future<Response> MockClientHandler(Request request);
```